ntp/syncTime.go

```diff
@@ -93,14 +94,12 @@ func NewSyncTime(ntpConfig config.NTPConfig, syncPeriod time.Duration, customQue
 }
 
 func checkNTPHost(ntpConfig config.NTPConfig, customQueryFunc func(options NTPOptions) (*ntp.Response, error)) int {
-	ntpConfig.Timeout = 10 * time.Millisecond
-
-	for hostIndex := range ntpConfig.Host {
+	for hostIndex := range ntpConfig.Hosts {
 		_, err := customQueryFunc(NewNTPOptions(ntpConfig, hostIndex))
 		if err != nil {
 			continue
 		}
-
+		log.Info(fmt.Sprintf("Using NTP server : %s", ntpConfig.Hosts[hostIndex]))
```

Comment:
use lowercase in the beginning of the log message

Reply:
done.